### PR TITLE
changing files to be single-file, structuring styles to have fallback…

### PR DESCRIPTION
**Pull request by sadiqevani**
This pull request deals with the issue of having two separate components repository. The new npm package `@aeternity/aepp-components-3` does not solve our issue and its slow in release and is a bit confusing.

I've turned the old components into a single `.vue` component so the `scss` in there gets bundled with the per-folder component, and also gets included in the `aepp.components.css` file so we don't need to use two different npm packages for the components.

@davidyuk I haven't tested on any application, and also we should have only one single npm package so we can do better transition.

### CODE REVIEW

File: src/components/aeAddressInput/aeAddressInput.vue [hunk 1]
```diff
@@ -8,7 +8,6 @@
     v-remove-spaces-on-copy
   />
 </template>
-
```
Comment (davidyuk) [inline src/components/aeAddressInput/aeAddressInput.vue:11]: It was intentional, I don't agree.

**Review by davidyuk — DISMISSED** [review]
All hex colors should be uppercase? Really? Check Zeplin or as we wrote before.

**Comment by davidyuk** [comment]
> The new npm package @aeternity/aepp-components-3 does not solve our issue

That issues with this package do we have? It works great in the Base app

File: src/components/aeAddressInput/aeAddressInput.vue (hunk 1 above)
Comment (sadiqevani) [reply src/components/aeAddressInput/aeAddressInput.vue:11]: What was intentional? The line break?

**Comment by sadiqevani** [comment]
@davidyuk in what sense it works great? This is not a long-term solution, are we suppose to have:

- `@aeternity/aepp-components-3`
- `@aeternity/aepp-components-4`
- `@aeternity/aepp-components-5` 

For each major version?

I know it works, its just not a proper solution! We will go through with proper release versions and we'll export old components through the releases to keep backward compatibility.

People working with components are kinda confused at what they should import and foremost some people don't know where are the new components.

File: src/components/aeAddressInput/aeAddressInput.vue (hunk 1 above)
Comment (davidyuk) [reply src/components/aeAddressInput/aeAddressInput.vue:11]: @sadiqevani yes

**Comment by davidyuk** [comment]
`@aeternity/aepp-components-3` is a temporary solution for the Base app, later we will be able to publish v3 at `@aeternity/aepp-components`

File: src/components/aeAddressInput/aeAddressInput.vue (hunk 1 above)
Comment (ricricucit) [reply src/components/aeAddressInput/aeAddressInput.vue]: I don't understand this review...is it about the linebreak or hex colors?

**Review by ricricucit — COMMENTED** [review]
I've tried, but this is too big of a PR for me to review without knowing the project very well. I also don't understand why there would be the need of having several `@aeternity/aepp-components` packages...I'd need more details to review...sorry.

**Review by ricricucit — APPROVED** [review]
👍

**Pull request merged**